test/testing_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'minitest/autorun'
+require 'spec_helper'
+
+describe "Testing" do
+  code = nil
+
+  before do
+    StripeMock.start
+
+    @observer = Class.new.tap do |cls|
+      cls.class_eval do
+        include Stripe::Callbacks
+      end
+    end
+
+    code = proc {|target, e| @event = e; @target = target}
+    @observer.class_eval do
+      after_invoice_payment_succeeded! do |evt, target|
+        code.call(evt, target)
+      end
+    end
+  end
+  
+  after do
+    ::Stripe::Callbacks.clear_callbacks!
+    StripeMock.stop
+  end
+
+  it "exposes a send_event method" do
+    (defined? Stripe::Testing.send_event).must_equal 'method'
+  end
+
+  it "forward the event to the registered callbacks" do
+    Stripe::Testing.send_event "invoice.payment_succeeded"
```

Comment:
The rspec style in `stripe-rails` follows the arrange, act, assert pattern. Anything method that is called for side-effects, should be inside a `before` block. So the following would look like:

```ruby
describe "when a payment succeeded event is sent" do
  before do
    Strip::Testing.send_event "invoice.payment_succeeded"
  end
  it "forwards the event to the registered callbacks" do
    @event.wont_be_nil
    @event.type.must_equal "invoice.payment_succeeded"
  end
end